src/containers/shared/utils.js

```diff
@@ -151,7 +151,10 @@ export const getLocalizedCurrencySymbol = (lang = 'en-US', currency = 'USD') =>
   return formatted.split('1')[0].trim();
 };
 
-export const formatLargeNumber = (d = 0, digits = 4) => {
+export const formatLargeNumber = (d = 0, digits = 6) => {
+  let digitsLocal = 6;
+  let zeros = 0;
+  let copy = d;
```

Comment:
This variable should be named more descriptively

Reply:
Done!